docs/rbac.md

```diff
@@ -0,0 +1,387 @@
+# RBAC V2
```

Comment:
Can we get a jargon table in here for things like inheritance, grants, and so on?